src/Form/DataPolicySettingsForm.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Drupal\gdpr_consent\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Class DataPolicySettingsForm.
+ *
+ * @ingroup gdpr_consent
+ */
+class DataPolicySettingsForm extends FormBase {
+
+  /**
+   * Returns a unique string identifying the form.
+   *
+   * @return string
+   *   The unique string identifying the form.
+   */
+  public function getFormId() {
+    return 'gdpr_consent_data_policy_settings';
+  }
+
+  /**
+   * Form submission handler.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
```

Comment:
@OH17L, just wondering. But could we not simply use `{@inheritdoc}` here (and also below)?

Reply:
@bramtenhove you are right. I will change it.